superset-frontend/src/explore/components/ExploreChartPanel.jsx

```diff
@@ -49,22 +52,124 @@ const propTypes = {
   triggerRender: PropTypes.bool,
 };
 
+export const EXPLORE_GUTTER_HEIGHT = 5;
```

Comment:
I'd be tempted to see if we can live without these constants, and use `gridUnit` instead (which may require `useTheme` in some cases).

Reply:
I'm not sure about that. We use `EXPLORE_GUTTER_MARGIN` both in component logic and in a styled component. In case we wanted to change its value, we'd have to remember to modify 2 places instead of 1. Unless I passed that margin value as a prop to styled component? We'd benefit from using a theme variable, but I think we'd lose some readability. WDYT?

Reply:
Maybe there's a compromise, setting the `EXPLORE_GUTTER_HEIGHT` and `EXPLORE_GUTTER_MARGIN` constants to the number of gridUnits, and then using that as the multiplier - in this case they'd both be set to `1`, I suppose. Which then raises the question of whether we need two consts or just use one unified gutter size const.

If this is all too much trouble or ruins the readability, maybe we can just change them both to `4` so they at least match the current world layout grid?

Reply:
I like this idea... but in this case 1px really does make a difference.
Here's a gutter height 4px:
![image](https://user-images.githubusercontent.com/15073128/101205061-7c51c180-366d-11eb-809e-7d8aa7b96aed.png)
And here's 5px:
![image](https://user-images.githubusercontent.com/15073128/101205090-883d8380-366d-11eb-8f85-4622451337c5.png)

Reply:
I pushed a change with `GUTTER_SIZE_FACTOR = 1.25`. Let me know what you think 🙂